uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/ui/NotificationTray.kt

```diff
@@ -95,18 +117,26 @@ class NotificationTray @Inject constructor(
             habit: Habit,
             notificationId: Int,
             timestamp: Timestamp,
-            reminderTime: Long
+            reminderTime: Long,
+            silent: Boolean = false
         )
 
         fun log(msg: String)
     }
 
-    internal class NotificationData(val timestamp: Timestamp, val reminderTime: Long)
-    private inner class ShowNotificationTask(private val habit: Habit, data: NotificationData) :
+    @Serializable
+    internal class NotificationData(
+        val timestamp: Timestamp,
+        val reminderTime: Long,
+    )
+
+    private inner class ShowNotificationTask(
+        private val habit: Habit,
+        private val data: NotificationData,
+        private val shown: Boolean
```

Comment:
`silent` is a bit more clear to me. Changing it would also make it consistent with the other class.

Reply:
Agree, done.